src/metadataGeneration/typeResolver.ts

```diff
@@ -86,11 +86,11 @@ export class TypeResolver {
     }
 
     if (this.typeNode.kind === ts.SyntaxKind.AnyKeyword) {
-      return { dataType: 'any' } as Tsoa.Type;
```

Comment:
While, strictly speaking, this is not necessary, it provides additional info. We are returning the bare base type, not some fancier type, like union, array, etc.